src/measure_theory/measure/finite_measure_weak_convergence.lean

```diff
@@ -279,13 +279,62 @@ end
 
 /-- Finite measures yield elements of the `weak_dual` of bounded continuous nonnegative
 functions via `finite_measure.test_against_nn`, i.e., integration. -/
-def to_weak_dual_bounded_continuous_nnreal (μ : finite_measure α) :
+def to_weak_dual_bcnn (μ : finite_measure α) :
   weak_dual ℝ≥0 (α →ᵇ ℝ≥0) :=
 { to_fun := λ f, μ.test_against_nn f,
   map_add' := test_against_nn_add μ,
   map_smul' := test_against_nn_smul μ,
   cont := μ.test_against_nn_lipschitz.continuous, }
 
+lemma to_weak_dual_bcnn_eval' (μ : finite_measure α) (f : α →ᵇ ℝ≥0) :
+  μ.to_weak_dual_bcnn f = μ.test_against_nn f := rfl
+
+lemma to_weak_dual_bcnn_eval (μ : finite_measure α) (f : α →ᵇ ℝ≥0) :
+  μ.to_weak_dual_bcnn f = (∫⁻ x, f x ∂(μ : measure α)).to_nnreal := rfl
+
+/-- The topology of weak convergence on `finite_measures α` is inherited (induced) from the weak-*
+topology on `weak_dual ℝ≥0 (α →ᵇ ℝ≥0)` via the function `finite_measures.to_weak_dual_bcnn`. -/
+instance : topological_space (finite_measure α) :=
+topological_space.induced
+  (λ (μ : finite_measure α), μ.to_weak_dual_bcnn) infer_instance
+
+lemma to_weak_dual_bcnn_continuous :
+  continuous (@finite_measure.to_weak_dual_bcnn α _ _ _) :=
+continuous_induced_dom
+
+/- Integration of (nonnegative bounded continuous) test functions against finite Borel measures
+depends continuously on the measure. -/
+lemma continuous_test_against_nn_eval (f : α →ᵇ ℝ≥0) :
+  continuous (λ (μ : finite_measure α), μ.test_against_nn f) :=
+(by apply (eval_continuous _ _).comp to_weak_dual_bcnn_continuous :
+  continuous ((λ φ : weak_dual ℝ≥0 (α →ᵇ ℝ≥0), φ f) ∘ to_weak_dual_bcnn))
+
+lemma tendsto_iff_weak_star_tendsto {γ : Type*} {F : filter γ}
+  {μs : γ → finite_measure α} {μ : finite_measure α} :
+  tendsto μs F (𝓝 μ) ↔
+    tendsto (λ i, (μs(i)).to_weak_dual_bcnn)
+      F (𝓝 μ.to_weak_dual_bcnn) :=
```

Comment:
Any reason this isn't on one line? This isn't important, and I don't think it technically goes against mathlib style guidelines, but personally it made more sense to me this way.
```suggestion
  tendsto μs F (𝓝 μ) ↔ tendsto (λ i, (μs(i)).to_weak_dual_bcnn) F (𝓝 μ.to_weak_dual_bcnn) :=
```

Reply:
The reason used to be that earlier I had the longer name `to_weak_dual_bounded_continuous_nonneg`, so these lines would have gone over 100 characters. After the name change I just didn't notice they now fit. Thanks, looks better this way!